### DISCUSSION TITLE
How do you use the colours?

### DISCUSSION THREAD

**Discussion by Random-Penguin** [fastfetch-cli/fastfetch#1469] [Q&A] [answered]
More specifically, I'm wondering how "{modifier}_" works. Do you add anything after it? Do you define another colour afterwards? I'm so confused...

**Comment by CarterLi** [comment, marked as answer]
`{modifier}_{standardcolor}`. For example, `bright_green`

See `fastfetch -h color` for detail

**Reply by Random-Penguin** [reply]
Ok, it was just weird how the schema didn't allow for that. Thanks for clearing that up!
